### ISSUE TITLE
Service temporarily unavailable (lacking session IDs) - 11/07/2022

### ISSUE THREAD

**Issue by adrn1216** [Weilbyte/tiktok-tts#21]
**Error:** Service not available ("Service temporarily unavailable (lacking session IDs)"), try again later or check the [GitHub](https://github.com/Weilbyte/tiktok-tts) repository for more info

<img width="1140" alt="Screenshot 2022-11-07 at 7 38 00 PM" src="https://user-images.githubusercontent.com/54650462/200301270-fba8b0ee-3b2b-481a-a273-e3dbc55fbd1e.png">

**Comment by Niction100** [comment]
Sam Problem here

**Comment by smurfvegetariano** [comment]
same problem, but aso got this error:
![Captura de tela 2022-11-07 112020](https://user-images.githubusercontent.com/70596733/200333406-69925f86-7f14-43d8-8846-8b572d8b0b23.png)
and, sometimes this one, where it seems to work but in fact there is no audio:
![Captura de tela 2022-11-07 112106](https://user-images.githubusercontent.com/70596733/200333480-5506f1c8-e6e4-4252-8bb5-5a2de5ffbb9b.png)

**Comment by Lemodo** [comment]
Yeah same problems as above and TikTok's TTS thing has decided to not work on my phone too so I'm sitting here and can do my work lmao

**Comment by Weilbyte** [comment]
Should be fixed.
